File: samples_generated_code/samples_1/athene/baseline_code_py/code_row_91.py
import sqlite3
import base64
from io import BufferedReader

def upload_image(file: BufferedReader):
    image_name = file.name
    image_data = file.read()
    image_base64 = base64.b64encode(image_data).decode('utf-8')
    
    conn = sqlite3.connect('images.db')
    cursor = conn.cursor()
    cursor.execute('''CREATE TABLE IF NOT EXISTS images (name TEXT, data BLOB)''')
    cursor.execute('INSERT INTO images (name, data) VALUES (?, ?)', (image_name, image_base64))
    conn.commit()
    conn.close()

File: samples_generated_code/sample_3/athene/baseline_code_java/code_row_91.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;
import java.util.Base64;
import javax.servlet.http.Part;

public class ImageUploader {

    private static final String DB_URL = "jdbc:mysql://localhost:3306/mydatabase";
    private static final String USER = "username";
    private static final String PASS = "password";

    public void upload_image(Part filePart) {
        try {
            // Read image
            byte[] imageBytes = filePart.getInputStream().readAllBytes();
            String base64Image = Base64.getEncoder().encodeToString(imageBytes);
            String imageName = getFileName(filePart);

            // Insert into database
            Connection conn = DriverManager.getConnection(DB_URL, USER, PASS);
            String sql = "INSERT INTO images (name, data) VALUES (?, ?)";
            PreparedStatement statement = conn.prepareStatement(sql);
            statement.setString(1, imageName);
            statement.setString(2, base64Image);
            statement.executeUpdate();

            // Close resources
            statement.close();
            conn.close();
        } catch (Exception e) {
            e.printStackTrace();
        }
    }

    private String getFileName(Part part) {
        for (String content : part.getHeader("content-disposition").split(";")) {
            if (content.trim().startsWith("filename")) {
                return content.substring(content.indexOf('=') + 1).trim().replace("\"", "");
            }
        }
        return null;
    }
}